### PR TITLE
Fix/reset feedback form bug

### PR DESCRIPTION
**Pull request by aslakihle**
[AB#443490](https://dev.azure.com/Equinor/4319082b-29ba-4111-a20b-c04f55c7e7c6/_workitems/edit/443490)

### CODE REVIEW

File: src/components/Navigation/TopBar/Help/Help.test.tsx [hunk 1]
```diff
@@ -406,10 +405,10 @@ describe('Help', () => {
       const button = screen.getByRole('button');
 
       await user.click(button);
-
       const reportBug = screen.getByText('Report a bug');
 
       await user.click(reportBug);
+      screen.logTestingPlaygroundURL();
```
Comment (AmandaElvkull) [inline src/components/Navigation/TopBar/Help/Help.test.tsx:411]: remove screen.log? 🙈

File: src/components/Navigation/TopBar/Help/Help.test.tsx [hunk 2]
```diff
@@ -591,8 +608,8 @@ describe('Help', () => {
 
       urlInput.blur();
       await waitForMS(1000);
-      const helperText = screen.queryByText(/URL must be from a equinor/i);
-
+      const helperText = screen.queryByText(/URL must be from a .equinor/i);
+      screen.logTestingPlaygroundURL();
```
Comment (AmandaElvkull) [inline src/components/Navigation/TopBar/Help/Help.test.tsx:612]: Same here, maybe remove screen.log ? 🙈

**Review by AmandaElvkull — CHANGES_REQUESTED** [review]

File: src/components/Navigation/TopBar/Help/Help.test.tsx (hunk 1 above)
Comment (aslakihle) [reply src/components/Navigation/TopBar/Help/Help.test.tsx]: You saw nothing 🤫 😂

File: src/components/Navigation/TopBar/Help/Help.test.tsx (hunk 2 above)
Comment (aslakihle) [reply src/components/Navigation/TopBar/Help/Help.test.tsx]: 🤫

**Review by AmandaElvkull — APPROVED** [review]

**Pull request merged**